tools/CMakeLists.txt

```diff
@@ -219,6 +240,21 @@ else()
     target_link_libraries(pcl_vtk2pcd vtkFiltersCore)
   endif()
 
+  PCL_ADD_EXECUTABLE(pcl_ply2obj COMPONENT ${SUBSYS_NAME} SOURCES ply2obj.cpp)
+  target_link_libraries(pcl_ply2obj pcl_io_ply)
+
+  PCL_ADD_EXECUTABLE(pcl_ply2ply COMPONENT ${SUBSYS_NAME} SOURCES ply2ply.cpp)
+  target_link_libraries(pcl_ply2ply pcl_io_ply)
+
+  PCL_ADD_EXECUTABLE(pcl_ply2raw COMPONENT ${SUBSYS_NAME} SOURCES ply2raw.cpp)
+  target_link_libraries(pcl_ply2raw pcl_io_ply)
+
+  PCL_ADD_EXECUTABLE(pcl_plyheader COMPONENT ${SUBSYS_NAME} SOURCES plyheader.cpp)
+  target_link_libraries(pcl_plyheader pcl_io_ply)
+
+  PCL_ADD_EXECUTABLE(pcl_converter COMPONENT ${SUBSYS_NAME} SOURCES converter.cpp)
+  target_link_libraries(pcl_converter pcl_common pcl_io)
```

Comment:
These don't need VTK (except for `pcl_converter`), but they are in the else-branch of `if(NOT VTK_FOUND)`?

Reply:
Yeah, I probably thought ply was part of VTK, but I geuss its not 😄 I'll move them accordingly.